recipe/meta.yaml

```
@@ -42,7 +42,7 @@ requirements:
   # See: https://github.com/man-group/ArcticDB/blob/master/environment_unix.yml
     - python
     - pip
-    - folly
+    - folly ==2023.09.25.00
```

The version of folly is already pinned there:

https://github.com/conda-forge/arcticdb-feedstock/blob/909370185cf2001a5a704e17d1dfcb589717a5b2/recipe/conda_build_config.yaml#L7-L9
and must not pinned here because the CI specification are not pinning folly.

For instance the re-rendering done with ba576fcc47b04ffe7f419d3fb14ab4222f4113d9 [just have unpinned folly](https://github.com/conda-forge/arcticdb-feedstock/pull/184/files#diff-f13aeb9d0aa59a69e1cdb52ddc65f19beab3c944be7295d9318605f7e220f7a5L23-L24).

This is one of the confusing quirks of conda-build, unfortunately. :confused: 

```suggestion
    - folly
```